packages/backend-common/src/database/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from './config';
+export * from './connection';
+export * from './postgres';
```

Comment:
There's no need to export anything but `createDatabase` right? Seems you can access all functionality through that

Reply:
@Rugvip - I thought about that as well, but since the underlying functions could in theory be used for further customization if needed. However, on second thought maybe we just start with them being private until an actual use case comes up for exposing them.

Reply:
@andrewthauer yes, exactly that, please make them private for now :grin:

I'm very interested in keeping API surface as slim as possible right now :>

Reply:
Done! I also added additional typing for the overrides to be `Partial<knex.Config>`.